### PR TITLE
feat(): squashtutorial

### PR DESCRIPTION
**Pull request by gilgda**
Reference #56

### CODE REVIEW

**Comment by gilgda** [comment]
Screenshots will be added on Friday

File: src/content/guides/GITHUB/SQUASH/SQUASH.MD [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿1. Öffne das `.gitconfig` Dokument, welches sich in `~/.gitconfig` befindet, oder öffne es einfach über das Terminal mit `git config --global -e`
```
Comment (DevVersion) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: I'm not sure about this,the changes will be lost after using another computer on our school network.

Comment (gilgda) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: Will they? Didn´t happen to me, but though I´m not sure I`d appreciate another solution if there even is one :flushed: :sweat_smile:

File: src/content/guides/GITHUB/SQUASH/SQUASH.MD [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: Please remove the unnessecary lines

File: src/content/guides/GITHUB/SQUASH/SQUASH.MD [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	```
+   
+	[alias]
+   
+	squash = "!f(){ git reset --soft HEAD~${1} && git commit --edit -m\"$(git log --format=%B --reverse HEAD..HEAD@{1})\"; };f"
+   
+	```
+   hinzu
+
+
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: Same as above

File: src/content/guides/GITHUB/SQUASH/SQUASH.MD [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+3. Speichere nun das Dokument; jetzt kann der Befehl `git squash [Anzahl der Commits]` genutzt werden
+
+
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: Same as above

File: src/content/guides/GITHUB/SQUASH/SQUASH.MD (hunk 2 above)
Comment (DevVersion) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: I think he kept these lines to make it visible, where the actual images will be placed.

Comment (Lordnoname) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: I think you look like this kid:
![https://gyazo.com/0957d076a6295ad42fa19eccb70daba1]

Comment (Lordnoname) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: Disregards - I didn't see his comment :+1:

**Comment by Lordnoname** [comment]
@DevVersion I think we should wait for him to add the screenshots before merging

File: src/content/guides/GITHUB/SQUASH/SQUASH.MD (hunk 1 above)
Comment (DevVersion) [inline src/content/guides/GITHUB/SQUASH/SQUASH.MD]: @Lordnoname Will create another PR which includes the permanent config tutorial. So your PR won't interfere with other contributions.

**Pull request closed without merging**